src/tests/eqrel_datastructure_test.cpp

```diff
@@ -260,22 +260,10 @@ TEST(DjTest, Scoping) {
     souffle::DisjointSet ds;
 }
 
-TEST(DjTest, MakeNode) {
+// A newly-created disjoint set has size 0.
+TEST(DjTest, SizeZero) {
     souffle::DisjointSet ds;
     EXPECT_EQ(ds.size(), 0);
-    souffle::block_t n = ds.makeNode();
-    // parent should be itself
-    EXPECT_EQ(DisjointSet::b2p(n), 0);
-    // rank should be 0
-    EXPECT_EQ(DisjointSet::b2r(n), 0);
-    EXPECT_EQ(ds.size(), 1);
-
-    souffle::block_t n2 = ds.makeNode();
-    // parent should be itself
-    EXPECT_EQ(DisjointSet::b2p(n2), 1);
-    // rank should be 0
-    EXPECT_EQ(DisjointSet::b2r(n2), 0);
-    EXPECT_EQ(ds.size(), 2);
```

Comment:
This test was removed because it is generalized/subsumed by one of the new property tests, same for `Clear`.